modules/ROOT/pages/kafka/kafka-connector-studio.adoc

```diff
@@ -19,92 +20,226 @@ To configure a connector in Anypoint Studio:
+** Topic Subscription Patterns (Optional) +
+The list of subscription regular expressions to subscribe to. Topics are automatically rebalanced according to the number of consumers of the topic.
+** Assignments (Optional) +
+The list of topic-partition pairs to assign. Note that there will be no automatic rebalance of the consumers.
 +
 image::kafka/kafka-consumer-studio-config.png[]
 +
-.. Producer operation:
-** Topic - Topic to send the message to.
-** Key - Key belonging to the message that is going to be sent.
-** Message - Message to be sent.
+** Default acknowledgment mode +
+Declares the type of acknowledgment mode supported
+** Default poll timeout +
+The amount of time to block. Specifies the total timeout for the polling.
+** Default poll timeout time unit +
+The time unit for the polling timeout. This combines with poll timeout to define the total timeout for the polling.
+** Zone ID (Optional) +
+Zone ID is used to convert the provided time stamps into `ZonedLocalDateTimes` in the results. Default value is the one provided by the system.
 +
-image::kafka/kafka-producer-studio-config.png[]
+image::kafka/kafka-consumer-studio-config-general.png[]
+
 
 == Configure an Input Source
 
-Configure an input source for the connector such as a connector operation, using an *HTTP Listener*, or *Scheduler*.
+Configure an input source for the connector such as the *Message Consumer* operation:
+
+[%header,cols="30s,70a"]
+|===
+| Name | Description
+| Configuration |The name of the configuration to use
+| Topic |Name of the topic from which to consume messages
+| Partition Offsets a| List of offsets (array of offsets) representing the partitions offsets configuration. For each element in the list you must specify a partition index and offset.
+| Primary Node Only |Whether to execute this source on only the primary node when running in a cluster.
+| Streaming Strategy a| * repeatable-in-memory-stream
+* repeatable-file-store-stream
+* non-repeatable-stream
+
+Configure to use repeatable streams.
+| Redelivery Policy a| The redelivery policy defines a policy for processing the redelivery of the same message.
+| Reconnection Strategy a| * reconnect
+* reconnect-forever +
+A retry strategy in case of connectivity errors
+|===
 
-The *Message Consumer* operation can be used as an input source in the Kafka connector.
 
-*Message Consumer* fields:
+The *Batch Message Consumer* operation can also be used as an input source in Kafka Connector:
 
 [%header,cols="30s,70a"]
 |===
 | Name | Description
 | Configuration |The name of the configuration to use.
-| Topic |Name of the topic to consume messages from.
-| Partition Offsets a| Array of Offset - List of Offset representing partitions offsets configuration. For each element in the list you have to specify partition index and offset.
-| Primary Node Only |Whether this source should only be executed on the primary node when running in Cluster. 
+| Poll timeout |The amount of time to block. 
```

Comment:
Throughout this topic set, go ahead and end each description with a period, since a few of them are followed by complete sentences.